lib/book-services.js

```diff
@@ -272,13 +272,16 @@ class BookServices {
     }
 
     /*
-        Step 6: Convert to .mobi
+        Step 7: Convert to .mobi
     */
 
     static convertToMobi(book) {
         return new Promise((resolve, reject) => {
-            exec(`${Config.KINDLEGEN} "${book.getEpubPath()}"`, (error) => {
-                if (error && error.code > 1) {
+            const kindlegenCommand = `${Config.KINDLEGEN} "${book.getEpubPath()}"`;
+            const calibreCommand = `calibre-bin/calibre/ebook-convert "${book.getEpubPath()}" "${book.getMobiPath()}" "--mobi-file-type=both" "--duplicate-links-in-toc"`;
```

Comment:
This is kinda minor - but looks like the path for `KINDLEGEN` is in the config, so this could probably go there too.

Reply:
As far as I can tell, the variable `Config.KINDLEGEN` is present because Kindlegen is, by default, in `/root/bin`, and the current platform needs to be set to decide which binary to use.

Calibre should be present in the current `WORKDIR`, which is why I decided to go with a relative path. I can set the path to an absolute of `/usr/src/epub-press/calibre-bin/calibre...` if you'd prefer that.

Reply:
Ah, gotcha. I actually think what you would have is:

```
Config.EBOOK_CONVERT = 'calibre-bin/calibre/ebook-convert';
```

Then that would get added here. Agree it's fragile though if things were called elsewhere with a different path. Perhaps we should add `calbre-bin` to the path?

eg.

```
PATH = $PATH:/usr/src/epub-press/calibre-bin/calibre
```

But maybe modifying the path has other risks. 

It's pretty minor. Like this is also fine.

Reply:
From what I can tell, Docker should always be using the WORKDIR of `/usr/src/epub-press`. Adding calibre to the path might work too, but as far as I can tell a relative path like this should be fine.

Maybe this would be a concern if epub-press is being run outside of Docker?